ht2/dz.css

```diff
@@ -0,0 +1,145 @@
+/*переподключить шрифт!!!!!!!!!!!!!*/
+body {
+	font-family: OpenSansRegular, serif, sans-serif;
```

Comment:
no, here should be stack for OpenSansRegular, so you should find smth similar. Sans-serif will never be applied as far as serif stands before. We should use only one of them. All fonts are separated into two families serif and sans-serif. So if OpenSans is from one - you should use it and only it.